integration-tests/modules/__tests__/stock-location/index.spec.ts

```diff
@@ -52,5 +52,46 @@ medusaIntegrationTestRunner({
         )
       })
     })
+
+    describe("Get stock location", () => {
+      let locationId
+      const location = {
+        name: "Test Location",
+      }
+      beforeEach(async () => {
+        const createLocationRespones = await api.post(
+          "/admin/stock-locations",
+          {
+            ...location,
+          },
+          adminHeaders
+        )
+        locationId = createLocationRespones.data.stock_location.id
+      })
+
+      it("should get a stock location", async () => {
+        const response = await api.get(
+          `/admin/stock-locations/${locationId}`,
+          adminHeaders
+        )
+
+        expect(response.status).toEqual(200)
+        expect(response.data.stock_location).toEqual(
+          expect.objectContaining({ id: locationId, ...location })
+        )
+      })
+
+      it("should get a stock location", async () => {
+        let error
+        await api
```

Comment:
comment: Same comment as seen on other PR. Rest API tests should live in `integration-tests/api`. 

Let's move forward with this PR and move later.